pino.d.ts

```diff
@@ -408,6 +408,14 @@ declare namespace pino {
          * The string key for the 'error' in the JSON object. Default: "err".
          */
         errorKey?: string;
+        /**
+         * The string key for the 'Request' in the JSON object. Default: "req".
+         */
+        requestKey?: string;
+        /**
+         * The string key for the 'Response' in the JSON object. Default: "res".
+         */
+        responseKey?: string;
```

Comment:
The key for request and response are now explicitly configurable. Previously, these had fixed values, set on pino-std-serializers ([mapHttpRequest](https://github.com/pinojs/pino-std-serializers/blob/master/lib/req.js#L98) and [mapHttpResponse](https://github.com/pinojs/pino-std-serializers/blob/master/lib/res.js#L45)).

Reply:
maybe this change makes the exported functions `mapHttpRequest` and `mapHttpResponse` deprecated, in [pino-std-serializers](https://github.com/pinojs/pino-std-serializers)?